src/ovp_pipeline/commands/_graph_visualizers.py

```
@@ -133,10 +139,13 @@ def _render_force_graph_section(
         f"viewBox='0 0 800 {GRAPH_VIEWPORT_HEIGHT}' "
         "preserveAspectRatio='xMidYMid meet' role='img' "
         f"aria-label='{escape(aria_label)}'>"
-        "<defs><marker id='cluster-arrow' viewBox='0 -5 10 10' "
-        "refX='15' refY='0' markerWidth='7' markerHeight='7' "
-        "orient='auto'><path d='M0,-5L10,0L0,5' fill='#9f9088'/>"
-        "</marker></defs>"
+        # No SVG <marker> arrowheads — they default to
+        # markerUnits='strokeWidth', which made a 7×7 triangle
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)